### DISCUSSION TITLE
第3节课作业(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#184] [作业]
请上传第三节作业在此处！

**Comment by Jianfeng777** [comment]
https://blog.csdn.net/jianfengkexin/article/details/135451548?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135451548%22%2C%22source%22%3A%22jianfengkexin%22%7D

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135478395?spm=1001.2014.3001.5502

**Comment by Alisa742** [comment]
https://blog.csdn.net/weixin_53222965/article/details/135474201

**Comment by zcg-joker** [comment]
[第三节课作业](https://blog.csdn.net/m0_55764641/article/details/135481142)

**Comment by Unicornlyy** [comment]
https://blog.csdn.net/m0_68165821/article/details/135551666

**Comment by l241025097** [comment]
第三节课作业
[03基于InternLM和LangChain搭建你的知识库.pdf](https://github.com/InternLM/tutorial/files/13920426/03.InternLM.LangChain.pdf)

**Comment by AxrLH** [comment]
第三次课作业
https://blog.csdn.net/Axr_Samuel/article/details/135551219?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135551219%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/677742342

**Comment by StarDuster** [comment]
https://gist.github.com/StarDuster/a23710c3afa1eef336638a426dfee940

**Comment by JACK626** [comment]
https://www.yuque.com/xiaotuanyue/mdr8d5/ul5cioe1v3hn6whp?singleDoc#
